crates/fj-math/src/point.rs

```diff
@@ -58,6 +58,11 @@ impl<const D: usize> Point<D> {
             coords: self.coords.to_xyz(),
         }
     }
+
+    /// Gives the distance between two Points.
```

Comment:
Nitpicky comment: I'd either use "point" as a normal word with proper capitalization, or make it clear I'm referring to the type `` `Point` ``, by adding backticks. And if we weren't already looking at the documentation of `Point`, I'd make it a link, ``[`Point`]``. If `Point` is in scope, rustdoc will make that link work automatically in the generated API reference).